genesis/process/shardGenesisBlockCreator.go

```diff
@@ -275,11 +276,7 @@ func createProcessorsForShardGenesisBlock(arg ArgsGenesisBlockCreator, enableEpo
 		Accounts:             arg.Accounts,
 		ShardCoordinator:     arg.ShardCoordinator,
 	}
-	builtInFuncFactory, err := builtInFunctions.NewBuiltInFunctionsFactory(argsBuiltIn)
-	if err != nil {
-		return nil, err
-	}
-	builtInFuncs, err := builtInFuncFactory.CreateBuiltInFunctionContainer()
+	builtInFuncs, err := builtInFunctions.NewBuiltInFunctionsFactory(argsBuiltIn)
```

Comment:
I think we need a renaming here. On the old code this returned a factory that we had to call CreateBuiltInFunctionContainer. Now this returns directly the container here

Reply:
done